eng/pipelines/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,44 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
+    type: string
+    default: ''  
+  - name: APIViewURL
+    type: string
+    default: 'https://apiview.dev' 
+  - name: StorageContainerUrl
+    type: string
+    default: ''
+
+variables:
+  PythonIndexUrl: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/'
+  PythonVersion: '3.10.2'
+  ApiStubVersion: '0.3.1'
+
+jobs:
+- job: CreatePythonReviewCodeFile
+  displayName: 'Create Python API review token file'
+  steps:
+  - template: /eng/pipelines/templates/variables/globals.yml
```

Comment:
This is a variable template not a steps template. I suspect this isn't doing what you want.